README.md

```diff
@@ -189,3 +108,12 @@ To deploy a branch from Heroku:
 3. Scroll all the way to the bottom to the **Manual Deploy** section.
 4. Choose a branch to deploy from the dropdown (it will most likely be the master branch).
 5. Click **Deploy Branch**.
+
+Note: If you add a Python package to the app, make sure you add that package to the **requirements.txt** file in the root directory of your version of the project.
```

Comment:
This should happen before deploying... maybe move up before the list of manual deploy steps?